pandas/tests/window/moments/test_moments_ewm.py

```diff
@@ -216,6 +216,9 @@ def _check_ew(self, name=None, preserve_nan=False):
         if preserve_nan:
             assert result[self._nan_locs].isna().all()
 
+    @pytest.mark.parametrize("min_periods", [0, 1])
```

Comment:
Can `_check_ew` be removed now? Or, I see it's still used in `test_ewma`...

Reply:
maybe i should rename `_check_ew` ? its indeed still used in `test_ewma`, `test_ewnvar`, `test_ewnvol` to conduct some type evaluation.

Reply:
see my comments on the merged PR. a lot of these could be module level (free) functions and have better names.